<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>500</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>700</width>
    <height>500</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>700</width>
    <height>500</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>AthloZone - Opciones</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../logoAthloZone.png</normaloff>../logoAthloZone.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget#centralwidget{
background-color: rgb(239, 81, 52);
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>30</y>
      <width>201</width>
      <height>191</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 36pt &quot;Franklin Gothic Demi Cond&quot;;
</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../logoAthloZone.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="botonSalirMenuOpciones">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>410</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(177, 5, 5);
font: 9pt &quot;MS Shell Dlg 2&quot;;
border-radius: 20px;
color: white
</string>
    </property>
    <property name="text">
     <string>Salir</string>
    </property>
   </widget>
   <widget class="QPushButton" name="botonRegresarMenuOpciones">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>410</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(177, 5, 5);
font: 9pt &quot;MS Shell Dlg 2&quot;;
border-radius: 20px;
color: white
</string>
    </property>
    <property name="text">
     <string>Regresar</string>
    </property>
   </widget>
   <widget class="QPushButton" name="botonControlarStock">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>240</y>
      <width>261</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 63 13pt &quot;Segoe UI Variable Display Semib&quot;;</string>
    </property>
    <property name="text">
     <string>Controlar Stock</string>
    </property>
   </widget>
   <widget class="QPushButton" name="botonRegistrarCompras">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>320</y>
      <width>261</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 63 13pt &quot;Segoe UI Variable Display Semib&quot;;</string>
    </property>
    <property name="text">
     <string>Registrar Compras</string>
    </property>
   </widget>
   <widget class="QPushButton" name="botonRealizarVentas">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>240</y>
      <width>261</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 63 13pt &quot;Segoe UI Variable Display Semib&quot;;</string>
    </property>
    <property name="text">
     <string>Realizar Ventas</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
